src/game/server/gamecontext.cpp

```diff
@@ -211,6 +212,14 @@ void CGameContext::Clear()
 	m_Tuning = Tuning;
 }
 
+void CGameContext::MakeLaserTextPoints(vec2 pPos, int pOwner, int pPoints){
```

Comment:
Could this function also exist in the controller? In ddnet-insta we try to avoid editing the gamecontext class to reduce git conflicts with ddnet. And allow downstream forks to overwrite the methods in their gamemodes.